### PR TITLE
your commit message

### PR DESCRIPTION
**Pull request by denis89**

### CODE REVIEW

File: #We Got Coders - My Amazing Application.html# [hunk 1]
```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
```
Comment (benlovell) [inline #We Got Coders - My Amazing Application.html#]: This file could be deleted from the repository. Feel free to remove it if you think it necessary.

File: We Got Coders - My Amazing Application.html~ [hunk 2]
```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
```
Comment (benlovell) [inline We Got Coders - My Amazing Application.html~]: Is this file of relevance? You should be mindful of what you're committing to git. If this is not needed, remember to delete it :+1:

File: We Got Coders - My Amazing Application.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
```
Comment (benlovell) [inline We Got Coders - My Amazing Application.html]: What was the intention with saving and committing this particular file? It seems like this was captured from running your submission and saving the result which wasn't strictly necessary, in my reviewing of your submission this is something I would do normally anyway :+1:

File: main.css [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+}
+
+.error {
+  text-align: center;
+}
+
+.error h2 {
+  font-size: 1.5em;
+}
+
```
Comment (benlovell) [inline main.css]: Try to apply your whitespace and indentation consistently. A consistent approach helps other developers scan and understand your code at a quick glance.

File: main.css~ [hunk 5]
```diff
@@ -0,0 +1,40 @@
+body {
```
Comment (benlovell) [inline main.css~]: It seems your editor may have been creating temporary files and that's why you've committed a few of these ones ending with the `~` character. If they're not useful to the submission you should try and remove them from the project.

File: prime.rb [hunk 6]
```diff
@@ -0,0 +1,3 @@
+require 'prime'
+
+puts Prime.first(100)
```
Comment (benlovell) [inline prime.rb]: How could you hook this up to the web application part of the submission? Remembering also that the challenge was to produce the sum of primes up to a particular number...

File: #We Got Coders - My Amazing Application.html# (hunk 1 above)
Comment (denis89) [inline #We Got Coders - My Amazing Application.html#]: Well, this file has got the most line. So, I don't think, it is a good idea to delete it.

Comment (denis89) [inline #We Got Coders - My Amazing Application.html#]: Meanwhile, I've find out, which files is relevant. So, I'm gonna delete this one.

File: main.css (hunk 4 above)
Comment (denis89) [inline main.css]: Right. Done.

File: main.css~ (hunk 5 above)
Comment (denis89) [inline main.css~]: Right. Also done.

File: prime.rb (hunk 6 above)
Comment (denis89) [inline prime.rb]: Right. I've updated this file meanwhile.

**Comment by denis89** [comment]
Done.

**Pull request closed without merging**